src/pages/Shop.tsx

```diff
@@ -0,0 +1,52 @@
+import Card from "../components/Card/Card";
+import Header from "../components/Header";
+import Drawer from "../components/Drawer/Drawer";
+import {useState, useEffect} from "react";
+
+type Item = {
+    title: string;
+    price: string;
+    imageUrl: string;
+}
+function Shop() {
+    const [items, setItems] = useState<Item[]>([]);
+    const [cartOpened, setCartOpened] = useState(false);
+    const [cartItems, setCartItems] = useState([]);
+
+
+    useEffect(() => {
+        fetch('https://640e020eb07afc3b0dbc8f12.mockapi.io/items').then(res => {
+            return res.json();
+        }).then(json => {
+            setItems(json)
+        })
+    }, []);
+
+    return (
+        <div className="wrapper clear">
+            {cartOpened && <Drawer onClickCart = {() => setCartOpened(false)} />}
```

Comment:
send cart items to drawer by props

```suggestion
            {cartOpened && <Drawer onClickCart = {() => setCartOpened(false)} items={cartItems} />}
```

Reply:
fixed